src/pytest_cases/case_parametrizer_new.py

```diff
@@ -205,19 +205,20 @@ def _glob_name_filter(case_fun):
     return _glob_name_filter
 
 
-def get_all_cases(parametrization_target,  # type: Callable
-                  cases=None,              # type: Union[Callable, Type, ModuleRef]
-                  prefix=CASE_PREFIX_FUN,  # type: str
-                  glob=None,               # type: str
-                  has_tag=None,            # type: Union[str, Iterable[str]]
-                  filter=None              # type: Callable[[Callable], bool]  # noqa
+def get_all_cases(parametrization_target=None,  # type: Callable
+                  cases=None,                   # type: Union[Callable, Type, ModuleRef]
+                  prefix=CASE_PREFIX_FUN,       # type: str
+                  glob=None,                    # type: str
+                  has_tag=None,                 # type: Union[str, Iterable[str]]
+                  filter=None                   # type: Callable[[Callable], bool]  # noqa
                   ):
     # type: (...) -> List[Callable]
     """
     Lists all desired cases for a given `parametrization_target` (a test function or a fixture). This function may be
     convenient for debugging purposes. See `@parametrize_with_cases` for details on the parameters.
 
-    :param parametrization_target: a test function
+    :param parametrization_target: a test function to get the module reference from. Required for cases that
```

Comment:
Here is where we could receive a test function or a module object (or even a module name, if we find a way to get it work). See my other comment [(in the "outdated" previous thread)](https://github.com/smarie/python-pytest-cases/pull/260#discussion_r811905945)